Add advanced theming (#1275)
streamlit/docs: content/develop/tutorials/theming/variable-fonts.md

   └── static/
       ├── NotoSans-Italic-VariableFont_wdth,wght.ttf
       ├── NotoSans-VariableFont_wdth,wght.ttf
       └── NotoSansMono-VariableFont_wdth,wght.ttf
   ```

1. To enable static file serving, in `.streamlit/config.toml`, add the following text:

   ```toml
   [server]
   enableStaticServing = true
   ```

   This makes the files in your `static/` directory publicly available through your app's URL at the relative path `app/static/{filename}`.

1. To define your alternative fonts, in `.streamlit/config.toml`, add the following text:

   ```toml
   [[theme.fontFaces]]
   family="noto-sans"
   url="app/static/NotoSans-Italic-VariableFont_wdth,wght.ttf"
   style="italic"
   [[theme.fontFaces]]
   family="noto-sans"
   url="app/static/NotoSans-VariableFont_wdth,wght.ttf"
   style="normal"
   [[theme.fontFaces]]
   family="noto-mono"
   url="app/static/NotoSansMono-VariableFont_wdth,wght.ttf"
   ```

   The `[[theme.fontFaces]]` table can be repeated to use multiple files to define a single font and to define multiple fonts. In this example, the definitions make `"noto-sans"` and `"noto-mono"` available to other font configuration options.

   <Tip>

>    For convenience, avoid spaces in your font family names. When you declare what font to use by default, you can also declare fallback fonts. If you avoid spaces in your font family names, you can avoid needing inner quotes.

Reviewer: ```suggestion
   For convenience, avoid spaces in your font family names. When you declare the default font, you can also declare fallback fonts. If you avoid spaces in your font family names, you don't need inner quotes.
```
